content/integrations/aws-secrets-manager.md

```diff
@@ -227,3 +225,128 @@ You need an [AWS access key and AWS secret key pair](https://docs.aws.amazon.com
      ciphertext: eyJhZWFkIjoiQUVTLTI1Ni1HQ00tSE1BQy1TSEEtMjU2IiwiaWQiOiIxMTc1ZjJjNDMyMjNjNjNmNjY1MDk5ZDExNmU3Yzc4NCIsIml2IjoiVHBtbHpWTDh5a2t4VVREV1RSTU5Tdz09Iiwibm9uY2UiOiJkeGl0R3A3bFB6S21rTE5HIiwiYnl0ZXMiOiJaaWdobEZrTUFuVVBWSG0wZDhSYUNBY3pnRWRsQzJqWFhCK1YxaWl2MXdnYjhBRytuTWx0Y3BGK0RtV1VoNkZaIn0=
    }
    ```
+
+## Using KES with a MinIO Server
+
+MinIO Server requires KES to enable server-side data encryption.
+
+See the [KES for MinIO instruction guide]({{< relref "/tutorials/kes-for-minio.md" >}}) for additional steps needed to use your new KES Server with a MinIO Server.
+
+
+## Configuration References
+
+The following section describes the Key Encryption Service (KES) configuration settings to use AWS Secrets Manager and AWS Key Management System as the root KMS for Server Side Encryption.
```

Comment:
So this is a ridiculous nuance, but strictly speaking KES doesn't perform SSE.

It just uses the root KMS for storage and management of External Keys, which are then used to produce cryptographic keys for use with *anything*. 

MinIO happens to use those keys for SSE, but there's no reason any other client could use it for anything else.

Reply:
replacing `for Server Side Encryption` with `to store external keys, such as the keys used for Server-Side Encryption on a MinIO Server` throughout.